BossMod/Modules/Shadowbringers/FATE/Formidable.cs

```diff
@@ -218,10 +223,48 @@ class SteamDome : Components.KnockbackFromCastTarget
         public SteamDome() : base(ActionID.MakeSpell(AID.SteamDome), 15) { }
     }
 
-    // TODO: stay/move component
     class DynamicSensoryJammer : Components.CastHint
     {
-        public DynamicSensoryJammer() : base(ActionID.MakeSpell(AID.DynamicSensoryJammer), "Stay still!") { }
+        public DynamicSensoryJammer() : base(ActionID.MakeSpell(AID.DynamicSensoryJammer), "") { }
+        private int ec;
```

Comment:
Should probably be a BitMask with bit set for each player with corresponding status - so eg. OnStatusGain would do `ec.Set(module.Raid.FindSlot(actor.InstanceID))`

Reply:
i dont know, it works as it is in replay files

Reply:
implemented it